content/doc/developer/testing/index.adoc

```diff
@@ -567,6 +567,13 @@ public class MyTest {
 }
 ----
 
+=== Verifying compatibility with Configuration as Code (JCasC)
+
+The link:https://plugins.jenkins.io/configuration-as-code/[Configuration as Code plugin] is an opinionated way to configure Jenkins based on human-readable declarative configuration files.
+Writing such a file should be feasible without being a Jenkins expert, just translating into code a configuration process one is used to executing in the web UI.
+If the plugin should support JCasC, it should also test the compatability.
```

Comment:
```suggestion
If the plugin supports JCasC, it also tests compatibility.
```
Suggestion to use present tense/avoid future voicing, update spelling. If this is incorrect, please disregard.